sc/social/like/testing.py

```diff
@@ -46,6 +46,7 @@ def setUpPloneSite(self, portal):
             self.applyProfile(portal, 'collective.cover:default')
 
         self.applyProfile(portal, 'sc.social.like:default')
+        portal.portal_workflow.setDefaultChain('simple_publication_workflow')
```

Comment:
isn't it supposed to break other tests where content are not published (like whatsapp and telegram robot tests)?